testutils/check-baselines.sh

```diff
@@ -28,47 +26,5 @@ Above baseline files contain string "$unhandledStr";
 some sample code won't render properly.
 
 EOF
-	exitStatus=1
-fi
-
-## Check that every calling form is used somewhere.
-callingFormsFile="src/main/java/com/google/api/codegen/viewmodel/CallingForm.java"
-
-# Calling forms are in an enum, so we look for lines that begin with whitespace plus a capital letter.
-callingForms=$(< $callingFormsFile grep -o '^[[:space:]]*[A-Z][a-zA-Z]*' | tr -d -c '[a-zA-Z\n]')
-
-unusedForms=$(
-	# Grep src/
-	#   recursively (-r)
-	#   in baseline files (--include='*.baseline')
-	#   for any calling forms: -f <(echo "$callingForms")
-	#   use fixed strings, not regexp (-F)
-	#   match whole word (-w)
-	#   print only the calling form, not surrounding line (-o)
-	#   don't print file names (-h)
-	# Prints all occurences of used calling forms
-	grep --include='*.baseline' -h -w -o -r -F -f <(echo "$callingForms") src/ |
-
-	# Then turn it into a set.
-	sort -u |
-
-	# Both $callingForms and the used forms determined so far in this pipeline are sets,
-	# so sorting  them together means
-	#   used forms appear twice together (one from $callingForms, one from the used forms)
-	#   unused forms appear only once
-	sort <(echo "$callingForms") - |
-
-	# Only print unique lines, leaving unused forms.
-	uniq -u
-)
-if [ "$unusedForms" ]; then
-	cat 1>&2 <<EOF
-Unused CallingForm:
-$unusedForms
-
-EOF
-	# There are unused calling forms right now, so commenting this off so we don't fail CI.
-	# exitStatus=1
+	exit 1
```

Comment:
This conditionally `exit 1` depending on whether or not the if-stmt fires. You might be confused by the indentation of the `EOF` line. Unfortunately herestring terminators must begin the line and whitespaces are not allowed.

Reply:
Oh, I see. We're still inside the `if` statement here. Yep, I parsed incorrectly. My bad. Carry on.

(I would have liked the diff to show the `fi`lines here matching, rather than below!)